doc/track_2_migration_guidance.md

```diff
@@ -0,0 +1,100 @@
+# Track 2 Migration Guidance
+
+Azure CLI is built upon Azure Python SDK. Recently Azure Python SDK announced next generation product. It is named Track 2 SDK. The old version of SDK is named Track 1. It claims that it has great advantages than Track 1 SDK. It is not compatible with Track 1 SDK. Azure CLI developers need to spend considerable time and do some work to migrate from Track 1 SDK to Track 2 SDK.
```

Comment:
suggest to put link of Track2 SDK if user interested at benefits from Track2 vs Track1.